### DISCUSSION TITLE
Getting error on GPTserver

### DISCUSSION THREAD

**Discussion by jigipatel7048** [acheong08/ChatGPT#547] [Q&A]
$**python3 -m GPTserver**
$/Library/Developer/CommandLineTools/usr/bin/python3: **No module named GPTserver**

I am getting errors on starting server where OfficialChatGPT and **python3 -m revChatGPT** working fine. Any input on setitngs?

**Comment by wzqstudio** [comment]
your revchatgpt may be install in your python3 path, not in the sitepackage path, add your revchatgpt path to PATH

**Reply by jigipatel7048** [reply]
@wzqstudio, installed Anaconda but still same error : 

**% python3 -m pip install revChatGPT**
Requirement already satisfied: revChatGPT in /opt/anaconda3/lib/python3.9/site-packages (1.1.4)
Requirement already satisfied: tiktoken in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT) (0.2.0)
Requirement already satisfied: openai in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT) (0.26.4)
Requirement already satisfied: aiohttp in /opt/anaconda3/lib/python3.9/site-packages (from openai->revChatGPT) (3.8.3)
Requirement already satisfied: tqdm in /opt/anaconda3/lib/python3.9/site-packages (from openai->revChatGPT) (4.64.1)
Requirement already satisfied: requests>=2.20 in /opt/anaconda3/lib/python3.9/site-packages (from openai->revChatGPT) (2.28.1)
Requirement already satisfied: blobfile>=2 in /opt/anaconda3/lib/python3.9/site-packages (from tiktoken->revChatGPT) (2.0.1)
Requirement already satisfied: regex>=2022.1.18 in /opt/anaconda3/lib/python3.9/site-packages (from tiktoken->revChatGPT) (2022.7.9)
Requirement already satisfied: lxml~=4.9 in /opt/anaconda3/lib/python3.9/site-packages (from blobfile>=2->tiktoken->revChatGPT) (4.9.1)
Requirement already satisfied: filelock~=3.0 in /opt/anaconda3/lib/python3.9/site-packages (from blobfile>=2->tiktoken->revChatGPT) (3.6.0)
Requirement already satisfied: urllib3<3,>=1.25.3 in /opt/anaconda3/lib/python3.9/site-packages (from blobfile>=2->tiktoken->revChatGPT) (1.26.11)
Requirement already satisfied: pycryptodomex~=3.8 in /opt/anaconda3/lib/python3.9/site-packages (from blobfile>=2->tiktoken->revChatGPT) (3.17)
Requirement already satisfied: idna<4,>=2.5 in /opt/anaconda3/lib/python3.9/site-packages (from requests>=2.20->openai->revChatGPT) (3.3)
Requirement already satisfied: charset-normalizer<3,>=2 in /opt/anaconda3/lib/python3.9/site-packages (from requests>=2.20->openai->revChatGPT) (2.0.4)
Requirement already satisfied: certifi>=2017.4.17 in /opt/anaconda3/lib/python3.9/site-packages (from requests>=2.20->openai->revChatGPT) (2022.9.24)
Requirement already satisfied: aiosignal>=1.1.2 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT) (1.3.1)
Requirement already satisfied: frozenlist>=1.1.1 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT) (1.3.3)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT) (4.0.2)
Requirement already satisfied: attrs>=17.3.0 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT) (21.4.0)
Requirement already satisfied: yarl<2.0,>=1.0 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT) (1.8.2)
Requirement already satisfied: multidict<7.0,>=4.5 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT) (6.0.4)

 % **python3 -m GPTserver**
**/opt/anaconda3/bin/python3: No module named GPTserver**



%**pip3 install "revChatGPT[unofficial]"**
Requirement already satisfied: revChatGPT[unofficial] in /opt/anaconda3/lib/python3.9/site-packages (1.1.4)
Requirement already satisfied: tiktoken in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT[unofficial]) (0.2.0)
Requirement already satisfied: openai in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT[unofficial]) (0.26.4)
Requirement already satisfied: tls-client>=0.1.7 in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT[unofficial]) (0.1.8)
Requirement already satisfied: undetected-chromedriver>=3.1.7 in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT[unofficial]) (3.4)
Requirement already satisfied: selenium>=4.7.2 in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT[unofficial]) (4.8.0)
Requirement already satisfied: 2captcha-python>=1.1.3 in /opt/anaconda3/lib/python3.9/site-packages (from revChatGPT[unofficial]) (1.1.3)
Requirement already satisfied: requests in /opt/anaconda3/lib/python3.9/site-packages (from 2captcha-python>=1.1.3->revChatGPT[unofficial]) (2.28.1)
Requirement already satisfied: trio-websocket~=0.9 in /opt/anaconda3/lib/python3.9/site-packages (from selenium>=4.7.2->revChatGPT[unofficial]) (0.9.2)
Requirement already satisfied: certifi>=2021.10.8 in /opt/anaconda3/lib/python3.9/site-packages (from selenium>=4.7.2->revChatGPT[unofficial]) (2022.9.24)
Requirement already satisfied: urllib3[socks]~=1.26 in /opt/anaconda3/lib/python3.9/site-packages (from selenium>=4.7.2->revChatGPT[unofficial]) (1.26.11)
Requirement already satisfied: trio~=0.17 in /opt/anaconda3/lib/python3.9/site-packages (from selenium>=4.7.2->revChatGPT[unofficial]) (0.22.0)
Requirement already satisfied: websockets in /opt/anaconda3/lib/python3.9/site-packages (from undetected-chromedriver>=3.1.7->revChatGPT[unofficial]) (10.4)
Requirement already satisfied: aiohttp in /opt/anaconda3/lib/python3.9/site-packages (from openai->revChatGPT[unofficial]) (3.8.3)
Requirement already satisfied: tqdm in /opt/anaconda3/lib/python3.9/site-packages (from openai->revChatGPT[unofficial]) (4.64.1)
Requirement already satisfied: blobfile>=2 in /opt/anaconda3/lib/python3.9/site-packages (from tiktoken->revChatGPT[unofficial]) (2.0.1)
Requirement already satisfied: regex>=2022.1.18 in /opt/anaconda3/lib/python3.9/site-packages (from tiktoken->revChatGPT[unofficial]) (2022.7.9)
Requirement already satisfied: filelock~=3.0 in /opt/anaconda3/lib/python3.9/site-packages (from blobfile>=2->tiktoken->revChatGPT[unofficial]) (3.6.0)
Requirement already satisfied: pycryptodomex~=3.8 in /opt/anaconda3/lib/python3.9/site-packages (from blobfile>=2->tiktoken->revChatGPT[unofficial]) (3.17)
Requirement already satisfied: lxml~=4.9 in /opt/anaconda3/lib/python3.9/site-packages (from blobfile>=2->tiktoken->revChatGPT[unofficial]) (4.9.1)
Requirement already satisfied: idna<4,>=2.5 in /opt/anaconda3/lib/python3.9/site-packages (from requests->2captcha-python>=1.1.3->revChatGPT[unofficial]) (3.3)
Requirement already satisfied: charset-normalizer<3,>=2 in /opt/anaconda3/lib/python3.9/site-packages (from requests->2captcha-python>=1.1.3->revChatGPT[unofficial]) (2.0.4)
Requirement already satisfied: async-generator>=1.9 in /opt/anaconda3/lib/python3.9/site-packages (from trio~=0.17->selenium>=4.7.2->revChatGPT[unofficial]) (1.10)
Requirement already satisfied: sortedcontainers in /opt/anaconda3/lib/python3.9/site-packages (from trio~=0.17->selenium>=4.7.2->revChatGPT[unofficial]) (2.4.0)
Requirement already satisfied: attrs>=19.2.0 in /opt/anaconda3/lib/python3.9/site-packages (from trio~=0.17->selenium>=4.7.2->revChatGPT[unofficial]) (21.4.0)
Requirement already satisfied: exceptiongroup>=1.0.0rc9 in /opt/anaconda3/lib/python3.9/site-packages (from trio~=0.17->selenium>=4.7.2->revChatGPT[unofficial]) (1.1.0)
Requirement already satisfied: outcome in /opt/anaconda3/lib/python3.9/site-packages (from trio~=0.17->selenium>=4.7.2->revChatGPT[unofficial]) (1.2.0)
Requirement already satisfied: sniffio in /opt/anaconda3/lib/python3.9/site-packages (from trio~=0.17->selenium>=4.7.2->revChatGPT[unofficial]) (1.2.0)
Requirement already satisfied: wsproto>=0.14 in /opt/anaconda3/lib/python3.9/site-packages (from trio-websocket~=0.9->selenium>=4.7.2->revChatGPT[unofficial]) (1.2.0)
Requirement already satisfied: PySocks!=1.5.7,<2.0,>=1.5.6 in /opt/anaconda3/lib/python3.9/site-packages (from urllib3[socks]~=1.26->selenium>=4.7.2->revChatGPT[unofficial]) (1.7.1)
Requirement already satisfied: multidict<7.0,>=4.5 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT[unofficial]) (6.0.4)
Requirement already satisfied: frozenlist>=1.1.1 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT[unofficial]) (1.3.3)
Requirement already satisfied: yarl<2.0,>=1.0 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT[unofficial]) (1.8.2)
Requirement already satisfied: aiosignal>=1.1.2 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT[unofficial]) (1.3.1)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in /opt/anaconda3/lib/python3.9/site-packages (from aiohttp->openai->revChatGPT[unofficial]) (4.0.2)
Requirement already satisfied: h11<1,>=0.9.0 in /opt/anaconda3/lib/python3.9/site-packages (from wsproto>=0.14->trio-websocket~=0.9->selenium>=4.7.2->revChatGPT[unofficial]) (0.14.0)
% **python3 -m GPTserver
/opt/anaconda3/bin/python3: No module named GPTserver**
